tuist/tuist
# Add support for specifying the RunAction `launchStyle`

Resolves https://github.com/tuist/tuist/issues/3511

### Short description 📝

Added a new property to RunAction called `launchStyle`

### How to test the changes locally 🧐

1. Generate a project with defined `launchStyle` for RunAction.
2. Go to `Edit scheme` -> `Run` and verify if the `Launch` On field is correctly set.

<img src="https://github.com/tuist/tuist/assets/74440939/ccb66266-1ca3-4cc1-a2a1-d451254f570e" width="500" />


### Contributor checklist ✅


### Reviewer checklist ✅
